### DISCUSSION TITLE
Modelando Domínios Ricos

### DISCUSSION THREAD

**Discussion by alxc89** [balta-io/forum#374] [Tira Dúvidas] [answered]
Olá, boa noite!

Fiz o curso 
Modelando Domínios Ricos e fiquei com uma dúvida sobre como utilizaria o domínio rico em uma API.
Como ligaria ela com o controller e outras camadas. Teria algum curso onde mostra isso? Criação do domínio rico junto com uma API?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @alxc89, como vai?
No caso seria só referenciar o projeto e consumir os handlers, acredito que mostro isto nos cursos mais avançados, inclusive no próximo que sai fim do mês!
